rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -1757,6 +1760,17 @@ def get_hashchainlog(entry):
         hashchain = id_to_hashchain[entry[0]]
         assert hashchain_to_address_mappings[hashchain] == entry[1:]
 
+    # Check Arbitrum One related data
+    assert cursor.execute(  # Check that Arbitrum One location was added
+        'SELECT location FROM location WHERE seq=?',
+        (Location.ARBITRUM_ONE.value,),
```

Comment:
```suggestion
        (Location.ARBITRUM_ONE.serialize_for_db(),),
```

Reply:
We select by seq here, which is the enum's value. Same as polygon check: 
```
assert cursor.execute(  # Check that Polygon POS location was added
        'SELECT location FROM location WHERE seq=?',
        (Location.POLYGON_POS.value,),
    ).fetchone()[0] == Location.POLYGON_POS.serialize_for_db()
```

Reply:
yeah this is wrong there. Location is a `DBCharEnumMixIn` and has DB serialization. Don't just copy paste without understanding.